apps/builder/app/shared/copy-paste/init-copy-paste.ts

```diff
@@ -77,6 +79,9 @@ export const initCopyPaste = (options: Options) => {
   };
 
   const handlePaste = (event: ClipboardEvent) => {
+    if (textEditingInstanceIdStore.get()) {
+      return;
+    }
```

Comment:
Found more solid solution. Here we already check contenteditable as invalid element. Though the problem is that lexical catches it earlier.

We can fixed it by capturing event. Add `{ capture: true}` for paste event.

Reply:
I tried, but the problem is when I get the event, the target will be already removed from the dom and replaced by lexical, so I can't properly detect contenteditable.

Reply:
let me try some more tricks

Reply:
Yes, I get this too with current event. Though capture worked before lexical removed element from dom.

Reply:
you were right, we even have that check, I must have somehow removed the capture or something